modules/analysis/rosbag-image-pipeline-sfn/coverage.ini

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+    tests/*
```

Comment:
This can be consolidated into `pyproject.toml`:

```toml
[tool.coverage.run]
branch = true
omit = [
  "tests/*"
]
```

That was you could also remove `--cov-config=coverage.ini` from `addopts`.